infra/terraform/modules/data_access/membership_events/membership_events.py

```diff
@@ -39,10 +34,15 @@ def event_received(sns_event, context):
         if event["scope"] != "team":
             continue
 
-        if action in EVENT_HANDLERS:
+        if action == "added":
             invoke_lambda(
-                function=EVENT_HANDLERS[action],
-                payload=payload(event),
+                function=os.environ["LAMBDA_ATTACH_BUCKET_POLICY_ARN"],
+                payload=attach_payload(event),
+            )
+        elif action == "removed":
+            invoke_lambda(
+                function=os.environ["LAMBDA_DETACH_BUCKET_POLICIES_ARN"],
+                payload=detach_payload(event),
```

Comment:
I think, rather than reintroducing the repetition that was factored out into `EVENT_HANDLERS`, you can just add the `policy` item to the payload `if action == "added"`